core/server/worker/src/main/java/alluxio/worker/netty/DataServerShortCircuitReadHandler.java

```diff
@@ -123,6 +123,9 @@ public Void call() throws Exception {
             if (mRequest.getPromote()) {
               try {
                 mWorker.moveBlock(mSessionId, mRequest.getBlockId(), mStorageTierAssoc.getAlias(0));
+              } catch (BlockDoesNotExistException e) {
+                LOG.debug("Block promotion skipped for {}: {}", mRequest.getBlockId(),
+                    e.getMessage());
```

Comment:
BlockDoesNotExistException can happen whenever reading from UFS. So it is fine to use debug. But when other exceptions (e.g. IOException, InvalidWorkerStateException etc) that can be thrown by moveBlock are thrown, something is wrong.